apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -255,6 +258,77 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
     expect(email.channel).to.equal(ChannelTypeEnum.EMAIL);
   });
 
+  it('should correctly set expiration date (TTL) for notification and messages', async function () {
```

Comment:
Would maybe split this to 3 test cases for each channel type it might be more easier to read. Wdyt?

Reply:
I think I will add a new test suite for this on Thursday, to also check with overrides and scheduled delay and multiple delayed actions in one workflow @scopsy